Client/src/Pages/Monitors/Details/index.jsx

```diff
@@ -267,7 +266,7 @@ const DetailsPage = () => {
                 </ButtonGroup>
               </Stack>
               <Box sx={{ height: "200px" }}>
-                <MonitorDetailsAreaChart checks={monitor.checks} />
+                <MonitorDetailsAreaChart checks={monitor.checks.reverse()} />
```

Comment:
Are we wanting to modify the original array here? 

I saw that this const was used before 

```javascript
const reversedChecks = monitor.checks.slice().reverse();
```

Reply:
Good catch, I don't think we want to mutate the array